src/features/common/WebappButton/index.tsx

```diff
@@ -84,7 +85,7 @@ function WebappButton({
 
     return (
       <Link
-        href={otherProps.href}
+        href={localizedPath(otherProps.href)}
```

Comment:
Since we're localizing paths at the WebappButton level, please check for double localization at places that use the WebappButton i.e. the locale is added to the href while passing it to WebappButton

For example, src/features/user/DonationReceipt/microComponents/ReceiptActions.tsx has

```
<WebappButton
            variant="secondary"
            text={tReceipt('modifyContactInformation')}
            elementType="link"
            target="_self"
            icon={<EditIcon />}
            href={`/${locale}/profile/donation-receipt/donor-contact-management`}
          />
```